x-pack/plugins/security_solution/cypress/integration/data_sources/sourcerer.spec.ts

```diff
@@ -10,37 +10,42 @@ import {
   loginWithUserAndWaitForPageWithoutDateRange,
 } from '../../tasks/login';
 
-import { HOSTS_URL } from '../../urls/navigation';
-import { waitForAllHostsToBeLoaded } from '../../tasks/hosts/all_hosts';
+import { HOSTS_URL, TIMELINES_URL } from '../../urls/navigation';
 import {
-  clickOutOfSourcererTimeline,
-  clickTimelineRadio,
+  addIndexToDefault,
+  deleteAlertsIndex,
   deselectSourcererOptions,
-  isCustomRadio,
+  isDataViewSelection,
   isHostsStatValue,
-  isNotCustomRadio,
+  isKibanaDataViewOption,
+  isNotSourcererOption,
   isNotSourcererSelection,
   isSourcererOptions,
   isSourcererSelection,
+  openAdvancedSettings,
+  openDataViewSelection,
   openSourcerer,
   resetSourcerer,
-  setSourcererOption,
-  unsetSourcererOption,
+  saveSourcerer,
+  waitForAlertsIndexToExist,
 } from '../../tasks/sourcerer';
+import { cleanKibana, postDataView } from '../../tasks/common';
 import { openTimelineUsingToggle } from '../../tasks/security_main';
-import { populateTimeline } from '../../tasks/timeline';
-import { SERVER_SIDE_EVENT_COUNT } from '../../screens/timeline';
-import { cleanKibana } from '../../tasks/common';
 import { createUsersAndRoles, secReadCasesAll, secReadCasesAllUser } from '../../tasks/privileges';
 import { TOASTER } from '../../screens/configure_cases';
+import { DEFAULT_ALERTS_INDEX, DEFAULT_INDEX_PATTERN } from '../../../common/constants';
+import { SOURCERER } from '../../screens/sourcerer';
 
 const usersToCreate = [secReadCasesAllUser];
 const rolesToCreate = [secReadCasesAll];
-// Skipped at the moment as this has flake due to click handler issues. This has been raised with team members
-// and the code is being re-worked and then these tests will be unskipped
+const siemDataViewTitle = 'Security Default Data View';
+const dataViews = ['auditbeat-*,fakebeat-*', 'auditbeat-*,beats*,siem-read*,.kibana*,fakebeat-*'];
+
 describe('Sourcerer', () => {
   beforeEach(() => {
```

Comment:
This can be a `before` instead of a `beforeEach` during the execution of the tests we don't modify any of the indexes we clean/create. Using `before` improves the execution time of the sorcerer tests:

With beforeEach:

<img width="405" alt="Screenshot 2021-11-22 at 14 47 38" src="https://user-images.githubusercontent.com/17427073/142873693-1251b4c5-3b98-47a9-8d2b-1ade566316cb.png">

With before:
<img width="402" alt="Screenshot 2021-11-22 at 14 41 36" src="https://user-images.githubusercontent.com/17427073/142873753-b033a2c3-e013-467b-a5b7-d22a540f5fa6.png">